src/test/java/com/jwplayer/southpaw/util/ByteArraySetTest.java

```diff
@@ -46,6 +73,636 @@ public ByteArraySet createSmallSet() {
         return set;
     }
 
+    public List<ByteArray> getRandomByteArrays(int count) {
+        return Stream.generate(() -> RandomStringUtils.randomAlphanumeric(RANDOM_STRING_SIZE))
```

Comment:
If we're concerned about reproducing potential test issues with randomly generated test data, we can consider seeding the random generator in tests. Showing a general idea here. Would recommend we generate a seed at the class level so it's the same for all tests in the run. We'd probably also want to be able to pass this in as variables at test time.
```suggestion
        final long seed = new Random().nextLong();
        System.out.println("Generating random data with seed: " + seed);
        return Stream.generate(() -> RandomStringUtils.random(RANDOM_STRING_SIZE, 0,0,true, true, null, new Random(seed)))
        return Stream.generate(() -> RandomStringUtils.randomAlphanumeric(RANDOM_STRING_SIZE))
```

Reply:
Should be fixed now